ci/kind/test-e2e-kind.sh

```diff
@@ -320,10 +321,15 @@ function run_test {
   coverage_args=""
   flow_visibility_args=""
 
+
+  # used for PacketCapture tests.
+  cat "$SFTP_DEPLOYMENT_YML" | docker exec -i kind-control-plane dd of=/root/sftp-deployment.yml
```

Comment:
personally I feel like the e2e test case itself should be in charge of creating the Deployment programmatically, using the K8s API. There is no need to make this script more complex than it is.

Reply:
yes, updated to use k8s api instead of yaml.